spec/controllers/between_categories_transactions_controller_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe BetweenCategoriesTransactionsController, type: :controller do
+  let(:user) { create(:user) }
+
+  before do
+    login_user user
+  end
+
+  describe 'GET #new' do
+    it 'renders the :new template' do
+      get :new
+      expect(response.status).to eq(200)
+    end
+  end
+
+  describe 'POST #create' do
+    context 'when valid' do
+      let(:amount) { BigDecimal.new(Faker::Number.decimal(3, 2)) }
+      let(:amount_from) { BigDecimal.new(10_000) }
+      let(:category_from) { create(:category, user: user, amount: amount_from) }
+      let(:category_to) { create(:category, user: user) }
+
+      let(:params) do
+        {
+          between_categories_transaction: {amount: amount, category_from_id: category_from.id,
+                                           category_to_id: category_to.id}
+        }
```

Comment:
It looks like format in this block isn't correct
```
{
  between_categories_transaction: 
    {
      amount: amount, 
      category_from_id: category_from.id,
      category_to_id: category_to.id
    }
}
```
Maybe this way it will look more clear

Reply:
or something like that:
```ruby
{
  between_categories_transaction: {
    amount: amount, 
    category_from_id: category_from.id,
    category_to_id: category_to.id
  }
}
```